### DISCUSSION TITLE
Handle large file uploads?

### DISCUSSION THREAD

**Discussion by Atomzwieback** [swoole/swoole-src#4585] [Q&A]
Hi,

is there a way to handle larger file uploads with swoole and if not why is it limited? We archived fileuploads up to 4GB but more than 4GB files running into an 400 - Bad Request error that is returned. Would be cool if anyone has a hint for us?

**Comment by he426100** [comment]
try [https://github.com/he426100/tus-php-hyperf](https://github.com/he426100/tus-php-hyperf)
